doc/source/whatsnew/v1.0.2.rst

```diff
@@ -48,6 +48,12 @@ Bug fixes
 
 .. ---------------------------------------------------------------------------
 
+.. _whatsnew_102.privacy.errors:
```

Comment:
move to 1.1, don't need a separate section, rather you can add the note to Other Enhancements. just say OptionError is now exposed in pandas.errors

Reply:
Done.